johnsonandjohnson/Bodiless-JS
# feat(components):  Add google tag manager &  datalayer.

## Changes
- Add google datalayer hoc.
- Add datalayer event component to pages and product detail pages. 

## Test Instructions
- Configure google tag manager in gatsby-config.js by adding the appropriate values.
- build and run the test site in production mode.
- Ensure the datalayer object is populated with the appropriate data. 

## Related Issues
Fixes #218
